source/com.google.firebase/firebase-perf/Transforms/Metadata.xml

```diff
@@ -12,6 +12,7 @@
     <attr path="/api/package[@name='com.google.firebase.perf.gauges']" name="managedName">Firebase.Perf.Gauges</attr>
     <attr path="/api/package[@name='com.google.firebase.perf.impl']" name="managedName">Firebase.Perf.Impl</attr>
     <attr path="/api/package[@name='com.google.firebase.perf.logging']" name="managedName">Firebase.Perf.Logging</attr>
+    <attr path="/api/package[@name='com.google.firebase.perf.transport']" name="managedName">Firebase.Perf.Transport</attr>
```

Comment:
https://github.com/xamarin/GooglePlayServicesComponents/pull/409/files/bc7cae92248fe27834de497d1b20841eec927eb8#diff-0d39a7c85a18a8136beec4f194f0c3c992a0578c02f4d1fc52600e30040643c3

Reply:
Is there something wrong with this metadata?